src/ypd/templates/project.html

```diff
@@ -28,6 +28,20 @@ <h1>{{project.title}}</h1>
         </div>
     {% endif %}
 
+    {% if project.can_be_modified_by(user) %}
+        <div class="projectButton">
+            <a href="{{url_for('ProjectView:archive', id=project.id, is_provided=(project is provided))}}">
+                <button>
+                    {% if project.archived == false %}
+                        🗑️
+                    {% else %}
+                        🔓 
+                    {% endif %}
+                </button>
+            </a>
+        </div>
+    {% endif %}
+
```

Comment:
You can only see this if you're a poster or an admin. 
What if an unrelated user looks at the project? Shouldn't they know it's been archived?

Reply:
Added an additional button(display only) that displays a lock icon if archive = true for all users to see.